core/client/src/main/java/alluxio/client/AbstractOutStream.java

```diff
@@ -40,7 +40,7 @@ public int getBytesWritten() {
   /**
    * Aborts the output stream.
    *
-   * @throws IOException if there is a failure to abort this output stream
+   * @throws IOException if an I/O error occurs
```

Comment:
I think we can relax the check for `@throws`, since a majority of these javadocs do not provide additional information. I would suggest to do this in another PR though.